source/_components/alarm_control_panel.homekit_controller.markdown

```diff
@@ -0,0 +1,16 @@
+---
+layout: page
+title: "HomeKit Alarm Control Panel"
+description: "Instructions how to setup HomeKit security systems within Home Assistant."
+date: 2018-12-27 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: apple-homekit.png
+ha_category: Alarm Control Panel
+ha_iot_class: "Local Polling"
+ha_release: 0.85
```

Comment:
We just cut 0.85 beta before the parent PR was merged, so this will go out in 0.86.